go/nbs/dynamo_manifest.go

```diff
@@ -39,13 +40,14 @@ type ddbsvc interface {
 type dynamoManifest struct {
 	table, db string
 	ddbsvc    ddbsvc
+	lock      addr
```

Comment:
re: the different ordering, i've seen that too. I think it's because, in `tableSet.Rebase()`, which we do whenever we open a manifest, we do the following to remove dupes in the set of tables we want to open:
```golang
	// Create a list of tables to open so we can open them in parallel.
	tablesToOpen := map[addr]tableSpec{}
	for _, spec := range specs {
		if _, present := tablesToOpen[spec.name]; !present { // Filter out dups
			tablesToOpen[spec.name] = spec
		}
	}
```
and then we iterate that map. Do you think it's important to fix the ordering? I'm sure we could come up with a way